src/pages/AdminDashboard/CreateRoomForm.tsx

```diff
@@ -29,17 +29,43 @@ import Flags from 'country-flag-icons/react/3x2'
 import { parsePhoneNumber } from 'libphonenumber-js'
 import { useTranslation } from 'react-i18next'
 
-const Button = styled(MuiButton)(spacing)
+export interface Values {
+    roomName: string
+    destinations: Array<any>
+    startAt: Date | null
+    hostName: string
+}
 
-const CreateRoomForm = ({ onFormSubmit }) => {
-    const { t } = useTranslation('dashboard')
-    interface Values {
-        roomName: string
-        participants: Array<any>
-        startAt: Date
+const mapDestinationsToInputField = (destinations) => {
+    const values = Array()
+    for (const destination of destinations) {
```

Comment:
simple suggestion
```
return destinations.reduce((accu, current) => (
  accu.concat(Object.values(current))), 
[]);
```

Reply:
I personally find `reduce` statements harder to read